packages/core-components/src/components/OAuthRequestDialog/OAuthRequestDialog.tsx

```diff
@@ -78,8 +81,10 @@ export const OAuthRequestDialog = () => {
         </List>
       </DialogContent>
 
-      <DialogActions>
-        <Button onClick={handleRejectAll}>Reject All</Button>
+      <DialogActions classes={{ root: classes.actionButtons }}>
+        <Button variant="contained" color="secondary" onClick={handleRejectAll}>
```

Comment:
Hmm, this one goes against [button hierarchy guidelines](https://material.io/components/buttons#hierarchy-and-placement) in Material design:

> Don't: Avoid using two contained buttons next to one another if they don’t have the same fill color.

This should be at most outlined, but probably makes sense to leave just as a text button since this is rarely going to be the desired action.

Reply:
Good point @timbonicus, thanks. I updated PR, and popup would look like this now 

<img width="568" alt="Screenshot 2021-09-13 at 09 06 37" src="https://user-images.githubusercontent.com/7230518/133041131-1bb7541c-b64c-4a15-83a8-903b627f9bc6.png">